cli/tools/coverage/mod.rs

```diff
@@ -58,9 +60,14 @@ pub struct CoverageCollector {
   session: LocalInspectorSession,
 }
 
-#[async_trait::async_trait(?Send)]
-impl crate::worker::CoverageCollector for CoverageCollector {
```

Comment:
It seems the trait was not being used, so I removed the trait. We can add it back in the future if we actually need it.

Reply:
That's fine